ember-bootstrap/addon/components/bs-contextual-help.ts

```diff
@@ -605,6 +605,7 @@ export default abstract class ContextualHelp<
     if (this.usesTransition) {
       transitionEnd(this.overlayElement, this.transitionDuration).then(
         tooltipHideComplete,
+        noop,
```

Comment:
What error do we expect here? Can we be more precise to only silence that one but not others? Maybe even catch it in the `tooltipHideComplete` action?